File: app/src/main/java/prince/sample/com/githubusers/data/remote/ApiService.java
package prince.sample.com.githubusers.data.remote;


import java.util.List;

import prince.sample.com.githubusers.model.User;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface ApiService {
    @GET("users")
    Call<List<User>> getUsers();

    @GET("users")
    Call<List<User>> getRemainingUsers(@Query("since") int userId);
}

File: kotlin/src/main/java/com/prince/githubusers/kotlin/data/remote/ApiService.kt
package com.prince.githubusers.kotlin.data.remote

import com.prince.githubusers.kotlin.model.User
import io.reactivex.Single
import retrofit2.http.GET
import retrofit2.http.Query


interface ApiService {
    @GET("users")
    fun getUsers(@Query("since") userId: Int): Single<List<User>>
}